From 3ae0b4508d34312338a3c5121c729b497d1a4a17 Mon Sep 17 00:00:00 2001
From: "junwei.ma" <junwei.ma@amlogic.com>
Date: Fri, 19 May 2023 17:40:55 +0800
Subject: [PATCH 1/6] ISP: MIPI camera sm1 ov5640 video0 to video50 [1/1]

PD#SWPL-124631

Problem:
T7 kernel update to 5.15

Solution:
T7 kernel update to 5.15

Verify:
ok

Change-Id: I06a7137a028e5a267aafb7a77870ad2add99845e
Signed-off-by: junwei.ma <junwei.ma@amlogic.com>
---
 drivers/media/camera/ov5640.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/camera/ov5640.c b/drivers/media/camera/ov5640.c
index c4591117..b4624ce3 100644
--- a/drivers/media/camera/ov5640.c
+++ b/drivers/media/camera/ov5640.c
@@ -80,7 +80,7 @@ MODULE_LICENSE("GPL v2");
 
 #define OV5640_DRIVER_VERSION "OV5640-COMMON-01-140717"
 
-static unsigned int video_nr = -1;
+static unsigned int video_nr = 50;
 
 static unsigned int debug;
 
-- 
2.25.1

